src/containers/shared/components/Transaction/OracleSet/parser.ts

```diff
@@ -0,0 +1,39 @@
+import { convertHexToString } from '../../../../../rippled/lib/utils'
+import { OracleSet } from './types'
+
+// Convert scaled price (assetPrice) to original price using formula:
+// originalPrice = assetPrice / 10**scale
+// More details: https://github.com/XRPLF/XRPL-Standards/discussions/129
```

Comment:
This should link to the published standard, not the discussion: https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-47d-PriceOracles